### PR TITLE
Log in for categorized application test (North Star)

### PR DESCRIPTION
**Pull request by devin-civi**
### Description

This changes a test to have a logged in user which a) More closely matches the non-northstar test and b) adds some screenshots with a logged in user.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [ ] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [ ] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [ ] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)

### Instructions for manual testing

If instructions are needed for manual testing by reviewers, include them here.

### Issue(s) this completes

Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

**Comment by devin-civi** [comment]
* **#7369** <a href="https://app.graphite.dev/github/pr/civiform/civiform/7369?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @devin-civi and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: browser-test/src/applicant/applicant_application_index.test.ts [hunk 1]
```diff
@@ -317,9 +318,13 @@ test.describe(
       )
     })
 
-    test('categorizes programs for draft and applied applications as guest user', async () => {
+    test('categorizes programs for draft and applied applications', async () => {
       const {applicantQuestions, page} = ctx
 
+      await disableFeatureFlag(page, 'north_star_applicant_ui')
```
Comment (dkatzz) [inline browser-test/src/applicant/applicant_application_index.test.ts:324]: **nitpick:** Maybe add a TODO to remove this disabling once login is handled in the north star work / and same below for logout

**Review by dkatzz — DISMISSED** [review]

File: browser-test/src/applicant/applicant_application_index.test.ts (hunk 1 above)
Comment (devin-civi) [reply browser-test/src/applicant/applicant_application_index.test.ts:325]: Done

File: browser-test/src/applicant/applicant_application_index.test.ts [hunk 2]
```diff
@@ -317,9 +318,14 @@ test.describe(
       )
     })
 
-    test('categorizes programs for draft and applied applications as guest user', async () => {
+    test('categorizes programs for draft and applied applications', async () => {
       const {applicantQuestions, page} = ctx
 
+      // TODO (https://github.com/civiform/civiform/issues/7377): Remove once we have a header with login/logout.
```
Comment (dkatzz) [inline browser-test/src/applicant/applicant_application_index.test.ts:324]: **suggestion:** Would you mind changing it to this format here and below: `TODO(#7377): Remove..`

**Review by swatkat1 — APPROVED** [review]

**Review by dkatzz — APPROVED** [review]
Meant to approve with my comment :)

**Review by dkatzz — APPROVED** [review]

File: browser-test/src/applicant/applicant_application_index.test.ts (hunk 2 above)
Comment (devin-civi) [reply browser-test/src/applicant/applicant_application_index.test.ts]: Done, thanks!

**Pull request merged**